Eastern-Research-Group/mywaterway
# HMW-856 Updated Protected Areas layer to new FeatureService

## Related Issues:
* [HMW-856](https://jira.epa.gov/browse/HMW-856)

## Main Changes:
* Switched to getting the PAD-US layer from its new home in ArcGIS Online.
  * The "new" layer is defined as a FeatureService rather than a MapImageLayer, which allows the removal of special handling that was previously necessary.

## Steps To Test:
1. Go to http://localhost:3000/community/dc/overview.
2. Turn on the "Protected Areas" layer. Verify it loads successfully.
3. Confirm the map legend shows Protected Areas items.
4. Click on a Protected Area on the map. Confirm the feature is highlighted and the proper popup is shown.
5. Print the map using the print widget. The layer and legend should render correctly on the printout.
